applications_tests/dem_3d/force_boundary_box.prm

```diff
@@ -0,0 +1,88 @@
+#---------------------------------------------------
+# Test
+#---------------------------------------------------
+subsection test
+    set enable 						 = true
+end
+
+# --------------------------------------------------
+# Model parameters
+#---------------------------------------------------
+subsection model parameters
+    set contact detection method   			                = dynamic
+    set dynamic contact search size coefficient             = 0.5
+    set neighborhood threshold				                = 1.4
+    set particle particle contact force method              = pp_nonlinear
+    set particle wall contact force method                  = pw_nonlinear
+    set rolling resistance torque method			        = constant_resistance
+    set integration method				                    = velocity_verlet
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+subsection physical properties
+    set gx            		 			            = 0
+    set gy            		 			            = 0
+    set gz            		 			            = -10
+    set number of particle types	                = 1
+    subsection particle type 0
+        set size distribution type		                = uniform
+        set diameter            	 	                = 0.005
+        set number				                        = 10
+        set density            	 		                = 2000
+        set young modulus particle         	            = 10000000
+        set poisson ratio particle          	        = 0.3
+        set restitution coefficient particle	        = 0.3
+        set friction coefficient particle               = 0.1
+        set rolling friction particle         	        = 0.05
+    end
+    set young modulus wall            			    = 10000000
+    set poisson ratio wall            			    = 0.3
+    set restitution coefficient wall           		= 0.3
+    set friction coefficient wall         		    = 0.1
+    set rolling friction wall         	      	  	= 0.05
+end
+
+#---------------------------------------------------
+# Insertion Info
+#---------------------------------------------------
+subsection insertion info
+    set insertion method				                = list
+    set list x = 0.0100, 0.0100, 0.0100, 0.0300, 0.0300, 0.0700, 0.0700, 0.0900, 0.0900, 0.0900
+    set list y = 0.0100, 0.0600, 0.0900, 0.0300, 0.0700, 0.0300, 0.0700, 0.0100, 0.0400, 0.0900
+    set list z = 0.0025, 0.0025, 0.0025, 0.0025, 0.0025, 0.0025, 0.0025, 0.0025, 0.0025, 0.0025
+    set insertion frequency            		 	        = 100
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+subsection mesh
+    set type                 				            = dealii
+    set grid type      	     				            = hyper_cube
+    set grid arguments       				            = 0:0.1:true
+    set initial refinement   				            = 4
+end
+
+subsection Forces and Torques
```

Comment:
We put subsection names in lower case letter and not capital
```suggestion
subsection forces and torques
```

Reply:
Additionally maybe a title like particles forces might be a better name. We always inherently assume that forces and torque go under the same section.

Reply:
I choose boundary forces because particle forces may be confusing